redcap_bridge/utils.py

```diff
@@ -129,25 +129,19 @@ def conversion_csv_to_json(csv_file):
     Test conversion function
     """
     df = pd.read_csv(csv_file)
-    df.to_json(
-        r'/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
-    json_df = pd.read_json(
-        '/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
-    for i in json_df.columns:
-        for j in json_df.index:
-            if json_df[i][j] == 'dropdown':
-                json_df[i][j] = dropdown_to_json()
-            elif json_df[i][j] == 'text':
-                json_df[i][j] = text_to_json()
-            elif json_df[i][j] == 'notes':
-                json_df[i][j] = notes_to_json()
-            elif re.search(r'@DEFAULT=.', str(json_df[i][j])):
-                print(f"MATCH")
-                json_df[i][j] = default_value_to_json()
-            else:
-                print(f"Other type")
-
-
+    json_df = df.to_dict()
+    for key, values in json_df.items():
+        if key == 'Field Type':
```

Comment:
Here you don't need to loop,but can directly `values['Field Type']`, since values here itself is a dictionary